Translation corrections (#434)
07th-mod/higurashi-console-arcs: Update/ura_009.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1054", 256, TRUE);
	OutputLine(NULL, "「仕切るお偉方にも説明すれば捜査員を一挙動員することは可能だし、まして県警の連中は君に好意的だ。",

> 			NULL, "\"If you explain the situation to the higher-ups, it'd be possible to act as an investigator, and in fact, you're on good terms with the prefectural police.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"If you explain the situation to the higher-ups, it'd be possible to act as an investigator. Plus, you're on good terms with the prefectural police.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1055", 256, TRUE);
	OutputLine(NULL, "彼らを活用して動いた方が効率もよく、何かと便利ではないのか？」",
			NULL, " Wouldn't it be more efficient and convenient to act through them?\"", Line_Normal);
	ClearMessage();

//巴rvS14/28/VT4Bb_tomoe1800.「ええ…実は、当初はそうするつもりで進めておりました。kvS14/28/VT4Bb_tomoe1801.…ですが」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1800", 256, TRUE);
	OutputLine(NULL, "「ええ…実は、当初はそうするつもりで進めておりました。",

> 			NULL, "\"Yes... in fact, I was planning to do that at first.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Yes... in fact, that's what I was planning to do at first.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1824", 256, TRUE);
	OutputLine(NULL, "「ですが、…勝手な期待になって申し訳ありませんが、山沖さんたちならきっと何らかの形で気づかれると思っていました」",

> 			NULL, "\"Still... I'm sorry for getting all these assumptions, but I figured you and the others would notice at some point anyhow.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Still... I'm sorry for making arbitrary assumptions, but I figured you and the others would notice at some point anyhow.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#7d746d>山沖</color>", NULL, "<color=#7d746d>Yamaoki</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1064", 256, TRUE);
	OutputLine(NULL, "「君には内密にと前置きした上で、私に頭を下げてきた。",
			NULL, "\"He bowed his head to me, asking me to keep this a secret from you.", Line_WaitForInput);
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1065", 256, TRUE);
	OutputLine(NULL, "どんな理由でもいい、南井警部が今回の別行動的な捜査から外れるよう説得してくれ、と」",

> 			NULL, " He told me to use whatever reason I could think of to persuade you to take you off this outside investigation.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " He told me to use whatever reasons I could think of and persuade you to take yourself off of this outside investigation.\"", Line_Normal);
```

			NULL, "\"You're just the chief of news correspondence, on a business trip on behalf of the Public Relations Office.", Line_WaitForInput);
	ModPlayVoiceLS(4, 30, "ps3/s14/30/vt4bb_yamaoki1086", 256, TRUE);
	OutputLine(NULL, "…肩書きは確かにそうだが、今君がいなくなれば、捜査本部はどうなるかね」",

> 			NULL, " ...It's true that it's just a title, but what do you think would happen to investigative HQ if we lost you now?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...It's true that it's just a title, but where do you think investigative HQ would be without you?\"", Line_Normal);
```
